agents/threadpool/hippo4j-agent-core/src/main/java/cn/hippo4j/agent/core/registry/AgentThreadPoolRegistry.java

```diff
@@ -19,48 +19,31 @@
 
 import cn.hippo4j.agent.core.logging.api.ILog;
 import cn.hippo4j.agent.core.logging.api.LogManager;
-import cn.hippo4j.common.config.ExecutorProperties;
 
 import java.util.Map;
 import java.util.Optional;
 import java.util.Properties;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ThreadPoolExecutor;
 
-public class AgentThreadPoolInstanceRegistry {
+public class AgentThreadPoolRegistry {
 
-    private static final ILog LOGGER = LogManager.getLogger(AgentThreadPoolInstanceRegistry.class);
+    private static final ILog LOGGER = LogManager.getLogger(AgentThreadPoolRegistry.class);
 
-    private final Map<String, AgentThreadPoolExecutorHolder> holderMap = new ConcurrentHashMap<>();
+    private static final Map<String, AgentThreadPoolExecutorHolder> holderMap = new ConcurrentHashMap<>();
 
-    public final Map<ThreadPoolExecutor, Class<?>> earlyConstructMap = new ConcurrentHashMap<>();
+    public static final Map<ThreadPoolExecutor, Class<?>> earlyConstructMap = new ConcurrentHashMap<>();
```

Comment:
Maybe you need to make this field private as well.
In addition, fields of type `static final` need to be named uppercase + underscore.

Reply:
> Maybe you need to make this field private as well. In addition, fields of type `static final` need to be named uppercase + underscore.

Thank you. I'll try to reconstruct him again